### PR TITLE
Enforce coding style on language

### PR DESCRIPTION
**Pull request by duncanspumpkin**

### CODE REVIEW

**Review by AaronVanGeffen — APPROVED** [review]

File: src/OpenLoco/Localisation/Languages.h [hunk 1]
```diff
@@ -3,7 +3,7 @@
 
 namespace OpenLoco::Localisation
 {
-    enum class loco_language_id : uint8_t
+    enum class LanguageId : uint8_t
```
Comment (AaronVanGeffen) [inline src/OpenLoco/Localisation/Languages.h:6]: I'd like to clarify that these are IDs from the original game. This list is therefore fixed.

Comment (duncanspumpkin) [reply src/OpenLoco/Localisation/Languages.h:6]: Hmm maybe they should be moved to the S5 namespace then

Comment (AaronVanGeffen) [reply src/OpenLoco/Localisation/Languages.h:6]: S5 doesn't actually deal with these IDs. It's just objects, as far as I'm aware.

Comment (duncanspumpkin) [reply src/OpenLoco/Localisation/Languages.h:6]: Yes I understand that but isn't the S5 namespace for vanilla structures/limits?

Comment (AaronVanGeffen) [reply src/OpenLoco/Localisation/Languages.h:6]: In my mind, it is only for the save format. I suppose more structs will be moved there eventually, though…

Personally, I'd rather see it as a `LegacyLanguageId` or something in one of the object (loader) headers.

Comment (duncanspumpkin) [reply src/OpenLoco/Localisation/Languages.h:6]: Okay will rename it

**Pull request merged**